device/core/src/twin.ts

```diff
@@ -21,18 +34,31 @@ export class Twin extends EventEmitter {
   static postEvent: string = 'post';
   static desiredPath: string = 'properties.desired';
 
+  /**
+   * The desired and reported properties dictionnaries (respectively in `properties.desired` and `properties.reported`).
+   */
   properties: any; // TODO: need type
 
   private _rid: number;
   private _client: Client;
   private _receiver: any; // TODO: need type
 
+  /**
+   * The constructor should not be used directly and instead the SDK user should use the {@link Client#getTwin} method to obtain a valid `Twin` object.
```

Comment:
>method to obtain a valid `Twin` object. [](start = 112, length = 39)

Although below you say gettwin is private.

Reply:
? Client.getTwin(), Registry.getTwin() and Twin.get() are all public.